test/avrora/storage/registry_test.exs

```diff
@@ -11,7 +11,7 @@ defmodule Avrora.Storage.RegistryTest do
   describe "get/1" do
     test "when request by subject name without version was successful" do
       Avrora.HTTPClientMock
-      |> expect(:get, fn url ->
+      |> expect(:get, fn url, _ ->
```

Comment:
Can you please add assert for the headers passed, otherwise who knows what is there?